patch/driver-arista-i2c-designware-shutdown.patch

```diff
@@ -0,0 +1,49 @@
+From: Justin Oliver <justinoliver@arista.com>
+Date: 2022-12-12 17:42:20 +0000
```

Comment:
Has this been sent upstream (especially for review)?

Reply:
Not yet ,but we could try submitting a patch for review to the Linux community.

Reply:
That’d be great. Please give a timeline. (It shouldn’t take longer than five minutes.)